javascript/javascript1/week3/SpeedCalculate.js

```diff
@@ -0,0 +1,17 @@
+
+  function calculateTravelTime(travelInfo){
```

Comment:
Nice on passing the info as a parameter to not depend on a globally defined variable :100: